base/Plain_netCDF_Time.F90

```diff
@@ -481,35 +482,34 @@ subroutine bisect_find_LB_R8_I8 (xa, x, n, n_LB, n_UB, rc)
        endif
        if( abs(klo-khi) <= 1 ) then
           n=klo
-          rc=0
-          return
+          status=0
+          exit
```

Comment:
Thank you for a lot of improvement throughout the PR, this is very helpful to me!   
For here, I think the deleted return is a logic return to the subroutine. I am leaving a note to myself to check back the actual run.